Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2223,11 +2171,27 @@ namespace OpenInfraPlatform {
 					double yRadius = ellipse->SemiAxis2 * UnitConvert()->getLengthInMeterFactor();
 
 					carve::geom::vector<3> ellipse_center =
-						conicPositionMatrix * carve::geom::VECTOR(0, 0, 0);
+						conicPositionMatrix * carve::geom::VECTOR(0., 0., 0.);
 
 					return ellipse_center + carve::geom::VECTOR(xRadius * cos(angle), yRadius * sin(angle), 0.);
 				}
 
+				/**********************************************************************************************/
+				/*! \brief Calculates a trimming point on the line using \c IfcParameterValue.
+				* \param[in] line					A pointer to data from \c IfcLine.
+				* \param[in] parameter				A pointer to data form \c IfcParameterValue.
+				* \return							The location of the trimming point.
+				*/
+				carve::geom::vector<3> getPointOnCurve(const EXPRESSReference<typename IfcEntityTypesT::IfcLine>& line,
+					const typename IfcEntityTypesT::IfcParameterValue & parameter)const throw (...)
+				{
+					double start_parameter = parameter * UnitConvert()->getLengthInMeterFactor();

```

Comment:
I believe that IfcLine is parameterized as [0,1] - please observe the documentation and correct it correspondingly.

Reply:
Resolved: c8ea968